pkg/cmd/clients/factory.go

```diff
@@ -747,6 +750,23 @@ func (f *factory) CreateHelm(verbose bool,
 	return h
 }
 
+func checkHelmCompatibility(helm *helm.HelmCLI) {
+	version, err := helm.VersionWithArgs(false, "--client")
+	version = strings.TrimPrefix(version, "Client: ")
+	if err != nil {
+		log.Logger().Warnf("Unable to detect the current helm version due to: %s", err)
+		return
+	}
+	v, err := semver.ParseTolerant(version)
+	if err != nil {
+		log.Logger().Warnf("Unable to parse the current helm version: %s", version)
+		return
+	}
+	if v.Major > 2 {
+		log.Logger().Fatalf("Your current helm version v3 is not supported. Please downgrade to helm v2.")
```

Comment:
Probably worth change to `Your current helm version v%d` with `v.Major`, just in case something goes truly insane and they release v4 while we're not looking. =)

Reply:
Fixed.